mne/simulation/tests/test_raw.py

```diff
@@ -110,6 +110,23 @@ def test_simulate_raw_sphere():
                              blink=True, ecg=True, random_state=seed,
                              use_cps=True)
     assert_array_equal(raw_sim_2[:][0], raw_sim[:][0])
+    std = dict(grad=2e-13, mag=10e-15, eeg=0.1e-6)
+    raw_sim = simulate_raw(raw, stc, trans, src, sphere,
+                           make_ad_hoc_cov(raw.info, std=std),
+                           head_pos=head_pos_sim, blink=True, ecg=True,
+                           random_state=seed, use_cps=True)
+    raw_sim_2 = simulate_raw(raw, stc, trans_fname, src_fname, sphere,
+                             cov=std, head_pos=head_pos_sim, blink=True,
+                             ecg=True, random_state=seed, use_cps=True)
+    assert_array_equal(raw_sim_2[:][0], raw_sim[:][0])
+    raw_sim = simulate_raw(raw, stc, trans, src, sphere,
+                           make_ad_hoc_cov(raw.info, std=None),
+                           head_pos=head_pos_sim, blink=True, ecg=True,
+                           random_state=seed, use_cps=True)
+    raw_sim_2 = simulate_raw(raw, stc, trans_fname, src_fname, sphere,
+                             cov='simple', head_pos=head_pos_sim, blink=True,
+                             ecg=True, random_state=seed, use_cps=True)
```

Comment:
Simulating these might take a long time. Can you see how much time it added to the tests?